cmd/describe/cluster/cmd.go

```diff
@@ -102,6 +102,29 @@ func run(cmd *cobra.Command, argv []string) {
 	var upgradeState *cmv1.UpgradePolicyState
 	var controlPlaneScheduledUpgrade *cmv1.ControlPlaneUpgradePolicy
 
+	// SDN -> OVN Migration Status
+	migrationsStr := ""
+	collection, err := r.OCMClient.FetchClusterMigrations(cluster.ID())
+	if err != nil {
+		r.Reporter.Warnf("Failed to get cluster migrations: %v", err)
+	}
+
+	migrationsToShow := map[string]string{}
```

Comment:
Any reason to have two maps instead of something like a map to an object of migrations?

Reply:
Parallel maps felt simpler and less code for the result here, those reasons would mainly be why I chose that, we don't particularly need all info from the migrations